Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/IBuildInfo.cs

```diff
@@ -57,6 +57,11 @@ public interface IBuildInfo
         /// </summary>
         ColorSpace? ColorSpace { get; set; }
 
+        /// <summary>
+        /// Optional parameter to set the scripting backend
+        /// </summary>
```

Comment:
Can you also add a remark that if nothing is provided, this will default to the current project settings?